presto-kafka/src/test/java/io/prestosql/plugin/kafka/TestKafkaIntegrationPushDown.java

```diff
@@ -13,41 +13,49 @@
  */
 package io.prestosql.plugin.kafka;
 
+import com.google.common.collect.ImmutableList;
```

Comment:
drop dot from commit message

We roughly try to follow https://chris.beams.io/posts/git-commit/